src/wasm-type.h

```diff
@@ -423,6 +416,7 @@ struct Field {
   } packedType; // applicable iff type=i32
   Mutability mutable_;
 
+  Field() = default;
```

Comment:
why?

Reply:
So I can write this without having to assign some arbitrary value first, which would reduce readability:

```
    Field field;
    if (lub(a.fields[i], b.fields[i], field)) {
      ...
```

Reply:
Makes sense, but it worries me a little. Does the C++ spec guarantee that the enums here will be initialized to their first value, or do we need an assignment for them?

Reply:
No, it leaves them uninitialized. I can add an explicit initializer so it at least has a valid value.